src/quickpicks/topics.ts

```diff
@@ -1,22 +1,35 @@
 import { ProgressOptions, ThemeColor, ThemeIcon, window } from "vscode";
-import { KafkaTopic } from "../models/topic";
-import { getSchemasForTopicEnv, getTopicsForCluster } from "../viewProviders/topics";
-import { KafkaCluster } from "../models/kafkaCluster";
 import { IconNames } from "../constants";
+import { Logger } from "../logging";
+import { KafkaCluster } from "../models/kafkaCluster";
+import { Schema } from "../models/schema";
+import { KafkaTopic } from "../models/topic";
+import { ResourceLoader } from "../storage/resourceLoader";
+
+const logger = new Logger("quickpicks.topics");
 
 export async function topicQuickPick(cluster: KafkaCluster): Promise<KafkaTopic | undefined> {
   const options: ProgressOptions = {
     location: { viewId: "confluent-topics" },
     title: "Loading topics...",
   };
   return window.withProgress(options, async () => {
-    const topics: KafkaTopic[] = await getTopicsForCluster(cluster, true);
-    if (topics.length === 0) {
-      window.showInformationMessage("No topics found in the cluster " + cluster.name);
-      return;
-    }
+    const loader = ResourceLoader.getInstance(cluster.connectionId);
+
+    // I ... guess we always want to force a refresh here?
+    // (was how @shoup did it in the original code)
+    const forceRefresh = true;
```

Comment:
I am not a fan of having the command-level driven quickpicks _always_ unilaterally bust caches. @shouples , howsabout a separate command expressly for global cache busting, so that then folks could run things like this in peace and they'd act like viewcontroller things -- showing cached stuff unless otherwise asked.

Reply:
I don't think we need a dedicated command for it, but I could see `forceRefresh` being an argument added to these kinds of quickpick functions if we're repeating the same return-cached-or-fetch pattern